src/main/groovy/com/github/benmanes/gradle/versions/reporter/HtmlReporter.groovy

```diff
@@ -0,0 +1,287 @@
+    if (!versions.isEmpty()) {
+      // The following dependencies exceed the version found at the '
+      //        + revision + ' revision level:
+      printStream.println("<H2>Later dependencies</H2>")
+      printStream.println("<p>The following dependencies have later ${revision} versions:<p>")
+      printStream.println("<table class=\"warningInfo\">")
+      getUpgradesRows(result).each { printStream.println it }
+      printStream.println("</table>")
+      printStream.println("<br>")
+    }
+  }
+
+  private def getUpgradesRows(Result result) {
+    List<String> rows = new ArrayList<>();
+    // The following dependencies are using the latest milestone version:
+    DependenciesGroup<DependencyOutdated> list = result.getOutdated()
+    rows.add("<thead><tr><th>Name</th><th>Group</th><th>URL</th><th>Current Version</th><th>Latest Version</th></tr></thead>")
+    for (DependencyOutdated item : list.dependencies) {
+      String rowString;
+      String rowStringFmt = "<tr><td>%s</td><td>%s</td><td>%s</td><td>%s</td><td>%s</td></tr>"
+      rowString = String.format(rowStringFmt, item.getName(), item.getGroup(),
+        getUrlString(item.getProjectUrl()), getVersionString(item.getGroup(), item.getName(), item.getVersion()),
+        getVersionString(item.getGroup(), item.getName(), item.getAvailable().getMilestone()))
+      rows.add(rowString)
+    }
+    return rows
+  }
+
+  private def writeUnresolved(printStream, Result result) {
+    SortedSet<DependencyUnresolved> versions = result.getUnresolved().getDependencies();
+    if (!versions.isEmpty()) {
+      //         '\nFailed to determine the latest version for the following dependencies '
+      //          + '(use --info for details):')
+      printStream.println("<H2>Unresolved dependencies</H2>")
+      printStream.println("<p>Failed to determine the latest version for the following dependencies:<p>")
+      printStream.println("<table class=\"warningInfo\">")
+      getUnresolvedRows(result).each { printStream.println it }
+      printStream.println("</table>")
+      printStream.println("<br>")
+    }
+  }
+
+  private def getUnresolvedRows(Result result) {
+    List<String> rows = new ArrayList<>();
+    // The following dependencies are using the latest milestone version:
+    DependenciesGroup<DependencyUnresolved> list = result.getUnresolved()
+    rows.add("<thead><tr><th>Name</th><th>Group</th><th>URL</th><th>Current Version</th></tr></thead>")
+    for (DependencyUnresolved item : list.dependencies) {
+      String rowString;
+      String rowStringFmt = "<tr><td>%s</td><td>%s</td><td>%s</td><td>%s</td></tr>"
+      rowString = String.format(rowStringFmt, item.getName(), item.getGroup(),
+        getUrlString(item.getProjectUrl()) , getVersionString(item.getGroup(), item.getName(), item.getVersion()))
+      rows.add(rowString)
+    }
+    return rows
+  }
+
+  private def writeGradleUpdates(printStream, Result result) {
```

Comment:
Since you are being fancy, you might want a convenience link to the Gradle release notes.

For an explicit version,
https://docs.gradle.org/$version/release-notes.html

For past releases,
https://gradle.org/releases/

Reply:
Next commit.